tests/cypress/integration/actions_projects_models/case_103_project_export.js

```diff
@@ -133,5 +134,23 @@ context('Export project dataset.', { browser: '!firefox' }, () => {
             cy.goToProjectsList();
             checkCounTasksInXML(exportAnnotationsRenameArchive, 1);
         });
+
+        it('Import dataset.', () => {
+            cy.openProject(projectName);
+            cy.deleteTask(task.name);
```

Comment:
@dvkruchinin, can you also add the case with no labels in the target project? Probably this case can be adjusted and the case of 3d tasks can remain unchanged.

Reply:
Sure. Added.